backend/database/database.py

```diff
@@ -5,11 +5,10 @@
 
 SQLALCHEMY_DATABASE_URL = f"mysql+pymysql://{os.environ['DATABASE_USER']}:{os.environ['DATABASE_PASSWORD']}@{os.environ['DATABASE_HOST']}/{os.environ['DATABASE_NAME']}"
 
+
```

Comment:
Extra newline?

Reply:
Yes! Fixed. fwiw one of the standards in python is double newline on class and method definitions, so, in some places it's desirable